sdk/core/core-tracing/src/instrumenter.ts

```diff
@@ -1,8 +1,13 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { Instrumenter, InstrumenterSpanOptions, TracingContext, TracingSpan } from "./interfaces";
-import { createTracingContext } from "./tracingContext";
+import {
```

Comment:
core-tracing is one of those modules that could bite us if someone loads CJS and ESM together since it uses the module scoped global variable. We should test this scenario as part of the testing party!

Reply:
It's worth testing, but a global scope should be a global scope regardless of what kind of module is being run.